docs/basics.md

```diff
@@ -0,0 +1,520 @@
+- `docker.io/myrepo/superimg:v1` &rarr; *rule 2* &rarr; *validator 2*(*key 2*): The image only matches rules 1 and 2 and thus 2 is chosen as it is more specific. That rule specifies `myvalidator` as validator but no trust root and thus validator 2 with trust root `default` (key 2) is used.
+- `docker.io/myrepo/myimg:v1` &rarr; *rule 3* &rarr; *validator 2*(*key 3*): The image matches all rules and thus 3 is chosen as it is most specific. The rule specifies `myvalidator` as validator with `mykey` as trust root and thus validator 2 with key 2 is used.
+
+Connaisseur ships with a few rules pre-configured.
+There is two rules that should remain intact in some form in order to not brick the Kubernetes cluster:
+
+- `k8s.gcr.io`: This is an `allow` rule for Kubernetes images (`k8s.gcr.io`) in order to not block cluster relevant images. These cannot be validated currently.
+- `docker.io/securesystemsengineering/*:*`: This rule is used to validate the Connaisseur images with the respective validator and removal can break the Connaisseur deployment. It is, however, possible to use the static `allow` validator.
+
+#### Configuration options
+
+`.policy[*]` in `helm/values.yaml` supports the following keys:
+
+| key  | default | required | description |
+| - | - | - | - |
+| `pattern` | - | :heavy_check_mark: | Globbing pattern to match an image name against. |
+| `validator` | `default` | | The name of a validator in the `validators` list. If not provided, the validator with name `default` is used if it exists. |
+| `with` | - | | Additional parameters to use for a validator. See more specifics in [validator section](validators/README.md). |
+|`with.trust_root`| `default` | | The name of a trust root, which is specified within the referenced validator. If not provided, the trust root with name `default` is used if it exists. |
+
+#### Example
+
+```yaml
+policy:
+- pattern: "*:*"
+- pattern: "docker.io/myrepo/*:*"
+  validator: myvalidator
+  with:
+    trust_root: mykey
+- pattern: "docker.io/myrepo/deniedimage:*"
+  validator: deny
+- pattern: "docker.io/myrepo/allowedimage:v*"
+  validator: allow
+```
+
+### Common examples
+
+Let's look at some useful examples for the `validators` and `policy` configuration.
+These can serve as a first template beyond the pre-configuration or might just be instructive to understand validators and policies.
+
+We assume your repository is `docker.io/myrepo` and a public key has been created.
+In case this repository is private, authentication would have to be added to the respective validator for example via:
+
+```yaml
+  auth:
+    secret_name: k8ssecret
+```
+
+The Kubernetes secret would have to be created separately according to the validator documentation.
```

Comment:
For the `instructive` example, I wouldn't link to `separately according to the validator documentation` but rather make it easy with username/pw or a yaml comment "contains uname+pw"